PhysicsTools/NanoAOD/plugins/VertexTableProducer.cc

```diff
@@ -74,6 +76,17 @@ class VertexTableProducer : public edm::stream::EDProducer<> {
   const std::string svName_;
   const std::string svDoc_;
   const double dlenMin_, dlenSigMin_;
+
+  typedef FuncVariable<reco::Vertex, StringObjectFunction<reco::Vertex>, int32_t> IntVar;
```

Comment:
here again, I'd rather not have this kind of things redefined outside of the templated table producer, why did you have to do this ?

Reply:
`FuncVariables` are defined inside `SimpleFlatTableProducerBase ` template. But, this producer is a custom table producer for vertices and to enable configurable optional parameters one needs to define instances of FuncVariable for `reco::Vertex` and various types (int32_t, float, etc.) somewhere. Is there better place to define those instances?